spec/vector_spec.rb

```diff
@@ -2009,6 +2009,13 @@ def expect_correct_tokens hash
     end
   end
 
+  context '#match' do
+      it 'returns matching array for a given regexp' do
+        dv = Daru::Vector.new ['3 days', '5 weeks', '2 weeks']
+        expect(dv.match /weeks/).to eq [nil, 2, 2]
+      end
```

Comment:
BTW, if I understand correctly, it is better to have only `false` and `true` values returned (though, if you'll follow `===` advice, you will have this automatically).